### PR TITLE
Style: Redesign contribution section

### PR DESCRIPTION
**Pull request by cp-priyanka-g**
**Note:** Testing done in few devices, As it is recreated previous UI and  tested design.
**Tested on:**
- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [x] Laptop / Macbook  (chrome, firefox, safari)
- [x] iPhone(safari)
- [x] Available android devices  (chrome, firefox)
- [x] iPad(safari)
- [x] Tablet (chrome, firefox)

### CODE REVIEW

File: vue-frontend/src/components/home-new/ContributionSectionMobile.vue [hunk 1]
```diff
@@ -0,0 +1,169 @@
+<template>
+  <div class="tw-container tw-justify-center tw-text-center">
+    <div class="tw-m-auto tw-w-[290px] sm:tw-w-full tw-pb-14 md:tw-pb-20">
+      <h1
+        class="tw-font-inter-bold tw-text-black-core/[.87] tw-text-[1.875rem] tw-leading-[2.5rem] md:tw-text-[3.4375rem] md:tw-leading-[5.5625rem]"
```
Comment (cp-jagruti-a) [inline vue-frontend/src/components/home-new/ContributionSectionMobile.vue:5]: i think we can keep it like tw-text-[1.875rem] md:tw-text-[3.4375rem] tw-leading-[2.5rem] md:tw-leading-[5.5625rem] for more readability

File: vue-frontend/src/components/home-new/ContributionSection.vue [hunk 2]
```diff
@@ -1,41 +1,41 @@
 <template>
   <div
-    class="tw-container lg:tw-flex lg:tw-flex-row-reverse tw-justify-center tw-max-w-[1878px] lg:tw-pt-20 tw-text-center lg:tw-text-left"
+    class="tw-relative xll:tw-container tw-my-0 tw-mx-auto tw-w-[90%] xll:tw-max-w-[1500px] tw-p-0"
```
Comment (cp-priyanka-g) [inline vue-frontend/src/components/home-new/ContributionSection.vue:3]: xll is working?

Comment (cp-sumi-k) [reply vue-frontend/src/components/home-new/ContributionSection.vue:3]: Yes, we had added this breakpoint in tailwind config

File: vue-frontend/src/components/home-new/PortfolioCTASection.vue [hunk 3]
```diff
@@ -41,7 +41,7 @@
             sleek solution.
           </div>
           <router-link
-            class="tw-flex tw-items-center tw-mx-auto sm:tw-mx-0 tw-w-max tw-rounded-full tw-p-3 tw-text-center tw-mr-2.5 tw-font-normal tw-text-[1rem] tw-leading-[1.1875rem] md:tw-text-[1.09375rem] md:tw-leading-[1.3125rem] lg:tw-text-[1.1875rem] lg:tw-leading-[1.4375rem] tw-font-inter-medium !tw-tracking-[0] tw-border-[1px] tw-border-solid tw-border-white tw-bg-white hover:tw-bg-transparent hover:tw-text-white active:tw-scale-[0.98] cta-button"
+            class="tw-flex tw-items-center tw-mx-auto lg:tw-mx-0 tw-w-max tw-rounded-full tw-p-3 tw-text-center tw-font-normal tw-text-[1rem] tw-leading-[1.1875rem] md:tw-text-[1.09375rem] md:tw-leading-[1.3125rem] lg:tw-text-[1.1875rem] lg:tw-leading-[1.4375rem] tw-font-inter-medium !tw-tracking-[0] tw-border-[1px] tw-border-solid tw-border-white tw-bg-white hover:tw-bg-transparent hover:tw-text-white active:tw-scale-[0.98] cta-button"
```
Comment (cp-jagruti-a) [inline vue-frontend/src/components/home-new/PortfolioCTASection.vue:44]: tw-w-max is this class is available

Comment (cp-sumi-k) [reply vue-frontend/src/components/home-new/PortfolioCTASection.vue:44]: Yes: its `width:max-content`

**Review by cp-jagruti-a — APPROVED** [review]

**Pull request merged**